Common/Source/Airspace/LKAirspace.cpp

```diff
@@ -1603,13 +1603,15 @@ void CAirspaceManager::FillAirspacesFromOpenAir(ZZIP_FILE *fp) {
                                     }
                                 }
                             }
-                            if (newairspace) {
+                            if(InsideMap) {
```

Comment:
if this condition is false, "newairspace" created line 1602 or 1596 is not added to _airspaces and pointer are set to null line 1629, and never deleted, so you have memory leak ...